ios/Classes/SwiftFlutterLoginFacebookPlugin.swift

```diff
@@ -234,6 +251,30 @@ public class SwiftFlutterLoginFacebookPlugin: NSObject, FlutterPlugin {
             result(data)
         }
     }
+
+    private func setAppId(result: @escaping FlutterResult, appId: AppId) {
+        let viewController = (UIApplication.shared.delegate?.window??.rootViewController)!
+        
+        _loginManager.setAppId(
```

Comment:
I can't see such method in `LoginManager`, but I can see `setAppId` in `FBSDKSettings` (`Settings` in Swift)
https://developers.facebook.com/docs/reference/ios/current/constants/FBSDKSettings/